lib/rbac.rb

```diff
@@ -1,4 +1,5 @@
 require 'rbac/filterer'
+require 'rbac/authorizer'
```

Comment:
I don't think these requires are necessary as Rails autoload should find them properly.  I only mention because it doesn't use require_dependency, so a change would need a restart of the whole server.  It's up to you how you want to organize.